src/screens/ChildrenScreen.js

```diff
@@ -1,158 +1,137 @@
-                    {data.note}
-                  </Text>
-                </TouchableOpacity>
-                <Ionicons
-                  name="ios-arrow-forward"
-                  size={30}
-                  color={colors.lightBlue}
-                  style={styles.arrowIcon}
-                ></Ionicons>
+            <View key={u.id}>
+              <View style={styles.container}>
+                {/* <Image source={{ uri:u.avatar }} /> */}
+                <Image
+                  style={styles.image}
+                  source={require("../../assets/icons/logo.png")}
+                />
+                <View style={styles.body}>
+                  <Text style={styles.name}>{u.name}</Text>
+                  <View style={styles.lineContainer}>
+                    <Image
+                      style={styles.icon}
+                      source={require("../../assets/icons/calendar.png")}
+                    />
+                    <Text style={styles.birth}>BIRTH: {u.birth}</Text>
+                  </View>
+                  <View style={styles.lineContainer}>
+                    <Image
+                      style={styles.icon}
+                      source={require("../../assets/icons/injection.png")}
+                    />
+                    <Text style={styles.birth}>LATEST: {u.latest}</Text>
+                  </View>
+                </View>
               </View>
             </View>
           );
         })}
       </View>
-    );
-  };
-  return (
-    <View style={screen.container}>
-      <ScrollView>{renderHistories()}</ScrollView>
-      <Ionicons
-        name="ios-add-circle"
-        size={90}
-        color={colors.lightBlue}
-        style={styles.addIcon}
-        onPress={() =>
-          navigation.navigate("AddChildScreen", { screen: "AddChildScreen" })
-        }
-      ></Ionicons>
+      <View style={styles.foot}>
+        <TouchableOpacity onPress={() => navigation.navigate("AddChildScreen")}>
+          <Image
+            style={styles.addIcon}
+            source={require("../../assets/icons/add-profile.png")}
+          ></Image>
+        </TouchableOpacity>
+      </View>
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  mainContainer: {
+    flex: 1,
+  },
+  header: {
+    flex: 0.8,
+  },
+  foot: {
+    flex: 0.2,
+  },
   container: {
-    width: "95%",
-    marginLeft: "5%",
-    marginTop: "2%",
-    paddingBottom: "2%",
-    borderBottomColor: colors.lightBlue_button,
-    borderBottomWidth: 1,
+    height: moderateScale(96),
+    backgroundColor: colors.white,
+    flexDirection: "row",
+    marginBottom: scale(10),
   },
-  listStyle: { paddingRight: "12%" },
-  type: {
-    fontSize: 20,
+  lineContainer: {
+    flexDirection: "row",
+  },
+  name: {
+    fontFamily: "notoSans-regular",
+    fontSize: moderateScale(16),
     fontWeight: "bold",
-    color: colors.lightBlue,
+    lineHeight: moderateScale(22),
+    margin: scale(5),
   },
```

Comment:
The start point of name and the icons (schedule and infection icons) should be the same. Currently, it doesn't like the same.